README.md

```diff
@@ -1,2 +1,41 @@
 # Awesome Book
 
```

Comment:
OPTIONAL
Add the project summary in the readme file. Remember, a professional repo includes a well detailed readme file.